models/marts/enriched_history/enriched_history_operations.sql

```diff
@@ -1,12 +1,17 @@
-{{ config(
-    tags = ["enriched_history_operations"],
-    materialized='incremental',
-    unique_key=["op_id"],
-    partition_by={
+{% set meta_config = {
+    "materialized": "incremental",
+    "unique_key": ["op_id"],
+    "cluster_by": ["ledger_sequence","transaction_id","op_account_id","type"],
+    "partition_by": {
         "field": "closed_at"
         , "data_type": "timestamp"
-        , "granularity": "month"},
-    cluster_by=["ledger_sequence","transaction_id","op_account_id","type"]
+        , "granularity": "month"}
+} %}
+
+{{ config(
+    tags=["enriched_history_operations"],
```

Comment:
nit: Why not include tags in the `meta_config` variable if you're unpacking **meta_config anyways?

Reply:
Yeah, I thought about same but did not have a preference. `tags` are anyways part of elementary schema. 
Can export tag as `meta_config` as well if you have a preference

Reply:
I'd prefer it in `meta_config` because that means the `set meta_config` block contains all the config info. And the actual config block only unpacks meta_config